js/ui/scheduler/appointmentDragBehavior.js

```diff
@@ -154,14 +162,16 @@ export default class AppointmentDragBehavior {
             immediate: false,
             onDragStart: this.onDragStart.bind(this),
             onDragMove: this.onDragMove.bind(this),
-            onDragEnd: this.onDragEnd.bind(this)
+            onDragEnd: this.onDragEnd.bind(this),
+            onDragCancel: this.onDragCancel.bind(this)
         }, config);
 
         this.appointments._createComponent(container, Draggable, extend({}, options, appointmentDragging, {
             onDragStart: this.createDragStartHandler(options, appointmentDragging),
             onDragMove: this.createDragMoveHandler(options, appointmentDragging),
             onDragEnd: this.createDragEndHandler(options, appointmentDragging),
             onDrop: this.createDropHandler(appointmentDragging),
+            onCancelByEsc: this.createCancelByEscHandler()
```

Comment:
Why do we need the ```createCancelByEscHandler``` method?

Maybe we can just set ```onCancelByEsc``` to ```true```

```suggestion
            onCancelByEsc: true
```